src/pages/index.tsx

```diff
@@ -64,13 +64,10 @@ export default function Home() {
     return (
         <Layout>
             <Hero
-                heading="Documentation"
+                heading="Apify Documentation"
```

Comment:
I know we discussed this many times that we don't want to repeat "Apify" on the landing page - Apify documentation, Apify SDK, Apify API Client ... The "Apify Docs" is already in the header

Reply:
In the title the APIFY logo + "Docs" is a small font, and it's easy to miss.

Plus see this:

<img width="1728" alt="Screenshot 2023-11-13 at 9 17 45" src="https://github.com/apify/apify-docs/assets/10612996/1ff915e8-2af5-446e-8de9-de0271ef91f5">
<img width="1725" alt="Screenshot 2023-11-13 at 9 17 37" src="https://github.com/apify/apify-docs/assets/10612996/941047c5-f9ea-4289-b20c-6d8ac754e9d2">
<img width="1728" alt="Screenshot 2023-11-13 at 9 17 32" src="https://github.com/apify/apify-docs/assets/10612996/690878c3-1b47-4d9f-860b-556fba3e8035">
<img width="1728" alt="Screenshot 2023-11-13 at 9 17 27" src="https://github.com/apify/apify-docs/assets/10612996/1c827a84-b98a-43e6-803d-9ac5aa41c2ba">
<img width="1728" alt="Screenshot 2023-11-13 at 9 15 57" src="https://github.com/apify/apify-docs/assets/10612996/933d5287-c267-4d59-a22e-2f2bc5482661">
<img width="1728" alt="Screenshot 2023-11-13 at 9 15 49" src="https://github.com/apify/apify-docs/assets/10612996/1e84c29f-5135-4160-9210-f80db1c2dc3c">
<img width="1728" alt="Screenshot 2023-11-13 at 9 15 42" src="https://github.com/apify/apify-docs/assets/10612996/12347acf-502a-4038-897e-67968be56963">

I'd keep "Apify Documentation", because that's what it is, and how we reference to it also externally.